### PR TITLE
chore: update locales for 1.7.0 release

### PR DESCRIPTION
**Pull request by scottdover**
**Summary**
This updates the translations needed for our 1.7.0 release.  There are two ways to update translations.
 - You can add suggestions for each translated string. [click here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) for more information on adding suggestions.
 - You can update translations using one of the csvs below by editing the file and re-attaching it to this pull request
    - **de**: [de-locale-update-1.7.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14141119/de-locale-update-1.7.0.csv)
    - **fr**: [fr-locale-update-1.7.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14141120/fr-locale-update-1.7.0.csv)
    - **it**: [it-locale-update-1.7.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14141121/it-locale-update-1.7.0.csv)
    - **ja**: [ja-locale-update-1.7.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14141122/ja-locale-update-1.7.0.csv)
    - **ko**: [ko-locale-update-1.7.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14141123/ko-locale-update-1.7.0.csv)
    - **pt-br**: [pt-br-locale-update-1.7.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14141124/pt-br-locale-update-1.7.0.csv)
    - **zh-cn**: [zh-cn-locale-update-1.7.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14141125/zh-cn-locale-update-1.7.0.csv)

**Testing**
 - [x] Verify all strings are captured by running the translation check (see https://github.com/sassoftware/vscode-sas-extension/actions/runs/7757082617)
 - [x] Verify **de** strings are translated
 - [x] Verify **fr** strings are translated
 - [x] Verify **ja** strings are translated
 - [x] Verify **ko** strings are translated
 - [x] Verify **zh-cn** strings are translated
 - [x] Verify **it** strings are translated
 - [x] ~Verify **pt-br** strings are translated~ (created a follow-up PR to handle these: https://github.com/sassoftware/vscode-sas-extension/pull/814)

### CODE REVIEW

**Comment by dfxvir** [comment]
[fr-locale-update-1.7.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14141739/fr-locale-update-1.7.0.csv)

**Comment by meilanji** [comment]
[ko-locale-update-1.7.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14155290/ko-locale-update-1.7.0.csv)

**Comment by Criptic** [comment]
@scottdover please find the updated German translations in this file
[de-locale-update-1.7.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14156986/de-locale-update-1.7.0.csv)

File: l10n/bundle.l10n.zh-cn.json [hunk 1]
```diff
@@ -32,11 +32,14 @@
   "Enter the URL": "输入网址",
   "Enter the URL for the SAS Viya server. An example is https://example.sas.com.": "输入SAS Viya服务器的URL。示例为 https://example.sas.com",
   "Enter the name of the SAS 9 SSH server.": "输入SAS 9 SSH服务器的名称。",
+  "Enter the name of the SAS 9 server.": "Enter the name of the SAS 9 server.",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:35]:
```suggestion
  "Enter the name of the SAS 9 server.": "输入SAS 9 服务器的名称。",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 2]
```diff
@@ -32,11 +32,14 @@
...
   "Enter the registered client ID. An example is myapp.client.": "输入注册的客户端ID。示例是 myapp.client。",
   "Enter the server name": "输入服务器名称",
   "Enter the server path": "输入服务器路径",
   "Enter the server path of the SAS Executable.": "输入SAS可执行文件的服务器路径。",
   "Enter your SAS server username.": "输入您的SAS服务器用户名。",
+  "Enter your password": "Enter your password",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:41]:
```suggestion
  "Enter your password": "输入您的密码",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 3]
```diff
@@ -32,11 +32,14 @@
...
+  "Enter your password for this connection.": "Enter your password for this connection.",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:42]:
```suggestion
  "Enter your password for this connection.": "输入此连接的密码。",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 4]
```diff
@@ -74,13 +77,15 @@
   "Result": "结果",
   "Result: {result}": "结果: {result}",
   "SAS 9 SSH Server": "SAS 9 SSH 服务器",
+  "SAS 9 Server": "SAS 9 Server",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:80]:
```suggestion
  "SAS 9 Server": "SAS 9 服务器",
```

File: package.nls.zh-cn.json [hunk 5]
```diff
@@ -39,6 +39,9 @@
   "configuration.SAS.connectionProfiles.profiles.connectionType": "SAS 配置文件连接类型",
   "configuration.SAS.connectionProfiles.profiles.context": "SAS Viya 上下文",
   "configuration.SAS.connectionProfiles.profiles.endpoint": "SAS Viya 连接配置文件端点",
+  "configuration.SAS.connectionProfiles.profiles.iom.host": "SAS IOM Connection Host",
```
Comment (scnwwu) [inline package.nls.zh-cn.json:42]:
```suggestion
  "configuration.SAS.connectionProfiles.profiles.iom.host": "SAS IOM 连接主机",
```

File: package.nls.zh-cn.json [hunk 6]
```diff
@@ -39,6 +39,9 @@
...
+  "configuration.SAS.connectionProfiles.profiles.iom.port": "SAS IOM Connection port",
```
Comment (scnwwu) [inline package.nls.zh-cn.json:43]:
```suggestion
  "configuration.SAS.connectionProfiles.profiles.iom.port": "SAS IOM 连接端口",
```

File: package.nls.zh-cn.json [hunk 7]
```diff
@@ -39,6 +39,9 @@
...
+  "configuration.SAS.connectionProfiles.profiles.iom.username": "SAS IOM Connection username",
```
Comment (scnwwu) [inline package.nls.zh-cn.json:44]:
```suggestion
  "configuration.SAS.connectionProfiles.profiles.iom.username": "SAS IOM 连接用户名",
```

File: package.nls.zh-cn.json [hunk 8]
```diff
@@ -50,6 +53,8 @@
   "configuration.SAS.flowConversionMode": "选择笔记本的转换模式",
   "configuration.SAS.flowConversionModeNode": "将每个笔记本单元转换为一个节点",
   "configuration.SAS.flowConversionModeSwimlane": "将每个笔记本单元转换为泳道",
+  "configuration.SAS.log.showOnExecutionFinish": "Show SAS Log when code execution is finished",
```
Comment (scnwwu) [inline package.nls.zh-cn.json:56]:
```suggestion
  "configuration.SAS.log.showOnExecutionFinish": "代码执行完成时显示 SAS 日志",
```

File: package.nls.zh-cn.json [hunk 9]
```diff
@@ -50,6 +53,8 @@
...
+  "configuration.SAS.log.showOnExecutionStart": "Show SAS Log when code execution starts",
```
Comment (scnwwu) [inline package.nls.zh-cn.json:57]:
```suggestion
  "configuration.SAS.log.showOnExecutionStart": "代码执行开始时显示 SAS 日志",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 10]
```diff
@@ -32,11 +32,14 @@
   "Enter the URL": "输入网址",
   "Enter the URL for the SAS Viya server. An example is https://example.sas.com.": "输入SAS Viya服务器的URL。示例为 https://example.sas.com",
   "Enter the name of the SAS 9 SSH server.": "输入SAS 9 SSH服务器的名称。",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:34]: Hi, it would be great to remove unused messages

**Comment by masayukiiizuka** [comment]
Attached JA file. Please review it
[ja-locale-update-1.7.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14159735/ja-locale-update-1.7.0.csv)

File: l10n/bundle.l10n.zh-cn.json (hunk 10 above)
Comment (scottdover) [reply l10n/bundle.l10n.zh-cn.json]: Thanks for the reminder :)

The locale tool now cleans these up, introduced in https://github.com/sassoftware/vscode-sas-extension/pull/793/commits/b6be8d8711d6e0e62d44300ad738e5fe19a6e89b.

The results of that are in https://github.com/sassoftware/vscode-sas-extension/pull/793/commits/df69c736a6f8486766f938cad121dc3a8dca0b47

Comment (scottdover) [reply l10n/bundle.l10n.zh-cn.json]: Made a couple more updates to do the same for package nls files - https://github.com/sassoftware/vscode-sas-extension/pull/793/commits/d0e2eeaef76bdfc7190285346745ea45872a676b / https://github.com/sassoftware/vscode-sas-extension/pull/793/commits/0e84ac7323c5ef07370a77bc1bedf71847faee03

**Comment by LRocc** [comment]
[it-locale-update-1.7.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14196577/it-locale-update-1.7.0.csv)

**Review by smorrisj — APPROVED** [review]

**Pull request merged**